README.md

```diff
@@ -1,101 +1,134 @@
 - Ask questions when you need clarification
 - Understand that the way you implement something may be different than the way your neighbor implements it
 
-## Getting Started
-This is a [level 3](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md), individual project.
-
-We will use the same project structure we used for the previous project. Classes should be in files in the `lib` folder, and tests should be in files in the `specs` folder. You should utilize a spec helper file. You will run tests by executing the `rake` command, as configured in a Rakefile. You should have **95% code coverage** using simplecov.
+**It is possible you will not be able to complete all requirements.** Though all 3 waves will be visible at the beginning of the project, they are organized by difficulty and relevance to the learning goals, and should be tackled in order.
 
 ### Setup
+
+We will use the same project structure we used for the previous project. Classes should be in files in the `lib` folder, and tests should be in files in the `specs` folder. You should utilize a spec helper file. You will run tests by executing the `rake` command, as configured in a Rakefile.
+
 1. Fork this repository in GitHub
 1. Clone the repository to your computer
 1. Create/copy a rakefile to run your tests
-1. Open the `specs/spec_helper.rb` file to load your classes and start up simple coverage.  This file will load all the required gems and source files your spec files need so they only need to require the helper.  
+1. Create the `specs/spec_helper.rb` file to load your classes and start `simplecov`. This file will load all the required gems and source files your spec files need so they only need to require the helper.
     - Each of your spec files should `require_relative` the spec helper file.
 1. Create a test to check the instantiation of one of your object types (**RED**)
 1. Create the class for the object tested in the step above (**GREEN**)
-1. Use git add, commit and push commands to push your initial code to GitHub
+1. Use `git add`, `commit` and `push` commands to push your initial code to GitHub
 
 ### Process
-You should use the following process as much as possible:  
+
+You should use the following process as much as possible:
 
 1. Write pseudocode
 1. Write test(s)
 1. Write code
+1. Commit
+
+You should have **95% code coverage** using `simplecov`.
+
+Your git commit history should provide a clear description of how your code developed, letting the reader know what changed when and why.
+
+## Wave Zero: Project Design
 
-Since this is the **last** Ruby-only project we are doing, and a level 3, the requirements are larger than previous projects. **It is possible you will not be able to complete all requirements**, but if you use the process above, you will still be able to share with your instructors the process and the design you have created.
+This will be an in-class activity.
```

Comment:
Does this refer to as the "OO Design" lecture?

If that's the case, do we need to rearrange introduction of things, so that it's more like:
1. OO Design general activity/lecture
2. Introduce Hotel
3. Do another OO Design activity that specifically accomplishes this Wave Zero?